src/components/Signin.js

```diff
@@ -0,0 +1,75 @@
+import '../styles/signinPage.scss';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { ToastContainer, toast, Zoom } from 'react-toastify';
+import { useTranslation } from 'react-i18next';
+import 'react-toastify/dist/ReactToastify.css';
+
+const LoginForm = ({ submitHandler, error, loading, details, setDetails }) => {
+  const isLogged = useSelector((state) => state.isLogged);
```

Comment:
I'm not seeing where we are using this `isLogged` function, can you explain why you introduced it?

Reply:
by now i am not using it, i introduced it to easy the work for the remaining tasks that will need successly loging in, but i can remove it and the one who need to redirect successfully logged user will need to declare it on their own